test/one/profiler/test/Runner.java

```diff
@@ -0,0 +1,284 @@
+        }
+
+        return true;
+    }
+
+    public static void run(Method m, Boolean retainLog) throws Exception{
+        for (Test test : m.getAnnotationsByType(Test.class)) {
+            if (!enabled(test)) {
+                log.log(Level.FINE, "Skipped " + m.getDeclaringClass().getName() + '.' + m.getName());
+                continue;
+            }
+            String testName = null;
+            if (retainLog) {
+                testName = m.getDeclaringClass().getName() + '.' + m.getName();
+            }
+
+            log.log(Level.INFO, "Running " + m.getDeclaringClass().getName() + '.' + m.getName() + "...");
+
+            try (final TestProcess p = new TestProcess(test, libExt, testName);) {
+                Object holder = (m.getModifiers() & Modifier.STATIC) == 0 ? m.getDeclaringClass().newInstance() : null;
+                m.invoke(holder, p);
+                log.info("OK");
+            } catch (Throwable e) {
+                System.out.println(e.getCause());
+                log.log(Level.WARNING, "Test failed " + e.getCause().getMessage(), e.getCause().getStackTrace());
+            }
+        }
+    }
+
+    public static void run(Class<?> cls, Boolean retainLog) throws Exception{
+        for (Method m : cls.getMethods()) {
+            run(m, retainLog);
+        }
+    }
+
+    private static boolean configureLogging(String[] args) {
+        if (System.getProperty("java.util.logging.ConsoleHandler.formatter") == null) {
+            System.setProperty("java.util.logging.ConsoleHandler.formatter", "java.util.logging.SimpleFormatter");
+        }
+        if (System.getProperty("java.util.logging.SimpleFormatter.format") == null) {
+            System.setProperty("java.util.logging.SimpleFormatter.format", "%4$s: %5$s%6$s%n");
+        }
+
+        String logLevelProperty = System.getProperty("logLevel");
+        if (logLevelProperty != null) {
+            Level logLevel =  Level.parse(logLevelProperty);
+            if (logLevel != null){
+                Logger.getLogger("").setLevel(logLevel);
+            } else {
+                Logger.getLogger("").setLevel(Level.INFO);
+            }
+        } else {
+            Logger.getLogger("").setLevel(Level.INFO);
+        }
+
+        String retainProfilesProperty = System.getProperty("retainProfiles");
+        if (retainProfilesProperty != null) {
+            if (retainProfilesProperty.equals("true")) {
+                return true;
+            } else if (retainProfilesProperty.equals("false")) {
+                return false;
+            } else {
+                return true;
+            }
+        } else {
+            return true;
+        }
+    }
+
+    public static void main(String[] args) throws Exception {
+        if (args.length == 0) {
+            System.out.println("Usage: java " + Runner.class.getName() + " TestName ...");
+            System.exit(1);
+        }
+
+        Boolean retainLogs = configureLogging(args);
```

Comment:
The return value for `configureLogging` is not obvious.
It's better to set some field instead. You can probably get `retainLogs` value in `TestProcess` static initializer, since `TestProcess` is the only place where this variable actually matters.